sds/src/standard_rules/mod.rs

```diff
@@ -0,0 +1,290 @@
+use crate::{Labels, MatchAction, RegexRuleConfig, Scope};
+use crate::{ProximityKeywordsConfig, RuleConfigTrait, SecondaryValidator};
+use serde::{Deserialize, Serialize};
+use std::fs;
+use std::fs::File;
+use std::io::BufReader;
+use std::path::Path;
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+pub struct StandardRule {
+    pub id: String,
+    pub name: String,
+    pub description: String,
+    pub pattern: String,
+
+    #[serde(default)]
+    pub included_keywords: Vec<String>,
+
+    #[serde(default)]
+    pub validators: Vec<StandardRuleValidator>,
+
+    /// Priority is not currently used by this library, but the data is available.
+    pub priority: u32,
+
+    /// Tags are not currently used by this library, but the data is available.
+    #[serde(default)]
+    pub tags: Vec<String>,
+
+    /// Labels are not currently used by this library, but the data is available.
+    #[serde(default)]
+    pub labels: Vec<String>,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+pub enum StandardRuleValidator {
+    LuhnChecksum,
```

Comment:
Why not adding the other already implemented ? (ChineseId and GitHub)

Reply:
Added.